packages/protocol/contracts/RequesterStore.sol

```diff
@@ -25,6 +25,14 @@ contract RequesterStore is IRequesterStore {
         _;
     }
 
+    function isEndorsed(uint256 requesterIndex, address addr)
```

Comment:
This is not needed because `requesterIndexToClientAddressToEndorsementStatus` is `public`

Reply:
oh i thought it wasn't for some reason ...

Reply:
I think it needs to be added to the interface as an external method